### DISCUSSION TITLE
Welcome to Adafruit_TinyUSB_Arduino Discussions!

### DISCUSSION THREAD

**Discussion by hathach** [adafruit/Adafruit_TinyUSB_Arduino#195] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DisasterAreaDesigns** [comment]
Hi folks!  I'm an engineer working on MIDI devices and guitar pedals.

**Comment by SmittyHalibut** [comment]
Hey all, Mark "Smitty" Smith, N6MTS here.  I develop hardware gadgets for Ham Radio stuff.

Currently, I want to add hardware flow control signals to the CDC profile, so the microcontroller can signal CTS, DSR, DCD, and RI to the host.  (See #491)

I'm currently trying to figure out how to setup a Adafruit_TinyUSB_Arduino dev environment in PlatformIO so I can hack at the TU code (to add functionality), and my code (to test that functionality), at the same time.  If anyone has any pointers of where to start (with the dev environment; I'm already pretty sure I know where to add the flow control signals), I'm all ears.

Cheers, 73 de N6MTS
-Mark
